dashboard/test/models/ability_test.rb

```diff
@@ -623,16 +623,15 @@ class AbilityTest < ActiveSupport::TestCase
     refute Ability.new(levelbuilder).can? :view_as_user, @login_required_script_level, student
   end
 
-  test 'student in same CSA code review enabled section as student seeking code review can view as peer' do
```

Comment:
should we add an additional test for students in same section but different code review groups cannot view as peer?

Reply:
mmm good idea, will add.